recipes/flask-multipass/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "flask-multipass" %}
+{% set version = "0.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-Multipass-{{ version }}.tar.gz
+  sha256: 536adcabb4db117556a8c92db71fb4df647d8ee3292dc5c39f444194c8f512b6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - authlib >=0.14.1,<=1.0.0a1
```

Comment:
Some of these dependencies are optional (extras in setup.cfg) - does conda have something similar?

Reply:
With conda you typically drag in the optional dependencies as well at runtime.

There is no real equivalent for extras AFAIK.